docs/_documentations/mdt-che-codechange.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: docs
+title: Making a code change with Codewind for Eclipse Che
+description: Making a code change with Codewind for Eclipse Che
+keywords: build, deploy, develop, public cloud, services, command line, cli, command, start, stop, update, open, delete, options, operation, devops
+duration: 1 minute
+permalink: mdt-che-codechange
+type: document
+---
+
+# Making a code change
+1. From the Explorer Projects view, find the project you want to edit and open the file you want to change.
+2. Edit the file.
+3. Save your changes.
+4. In the Codewind Project Explorer view, Codewind detects the change and updates the build status. Codewind builds and deploys the project to incorporate your update.
```

Comment:
This last sentence isn't entirely accurate, I'd say. Better to say something like "Codewind will detect the change and rebuild the project. The project's application and build status will be updated in the Codewind Project Explorer view"